### PR TITLE
Update license to EPL 2.0

### PR DESCRIPTION
**Pull request by sunix**
https://github.com/eclipse/che/issues/10465

### CODE REVIEW

**Comment by benoitf** [comment]
if you want to enforce the rule: https://github.com/eclipse/theia-generator-plugin/blob/master/tslint.json#L3-L7

**Comment by sunix** [comment]
I guess I can reuse this tslint.json file for all our project

File: package.json [hunk 1]
```diff
@@ -3,10 +3,14 @@
   "scripts": {
     "prepare": "lerna run prepare",
```
Comment (benoitf) [inline package.json]: maybe have tslint in prepare or build task ?

File: package.json [hunk 2]
```diff
@@ -3,10 +3,14 @@
...
     "rebuild:browser": "theia rebuild:browser",
-    "rebuild:electron": "theia rebuild:electron"
+    "rebuild:electron": "theia rebuild:electron",
+    "tslint-fix": "tslint --fix --project github-extension",
+    "tslint": "tslint --fix --project github-extension"
```
Comment (benoitf) [inline package.json]: in your example I don't see differences between tslint and tslint-fix

File: package.json (hunk 1 above)
Comment (sunix) [reply package.json]: done

File: package.json (hunk 2 above)
Comment (sunix) [reply package.json]: done: removed --fix in the tslint one

**Review by benoitf — APPROVED** [review]

**Pull request merged**